tests/crt/aot_memory_test.cc

```diff
@@ -24,83 +25,118 @@
 
 // Check with LIFO checks enabled for stack allocator
 #define TVM_CRT_STACK_ALLOCATOR_ENABLE_LIFO_CHECK
+
+/*!
+ * Align memory pointer.
+ * \return Number of memory offset.
+ */
+uint32_t memory_align(uint8_t** memory_ptr) {
```

Comment:
make this a static function and consider calling this `align_pointer`, also document that `memory_ptr` is modified in the docstring.